devops/scripts/register_all_bundles_in_repo.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Should this file be named publish_and_register_all_bundles.sh?
plus, maybe add a make target `bundle-publish-register-all`?

Reply:
Done